src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.tsx

```diff
@@ -1,23 +1,45 @@
+import type {ForwardedRef} from 'react';
 import React, {forwardRef, useCallback, useEffect, useImperativeHandle, useRef, useState} from 'react';
+import type {EmitterSubscription, GestureResponderEvent, NativeTouchEvent, View} from 'react-native';
 import {Dimensions} from 'react-native';
-import _ from 'underscore';
+import type {OnyxEntry} from 'react-native-onyx';
 import ConfirmModal from '@components/ConfirmModal';
 import PopoverWithMeasuredContent from '@components/PopoverWithMeasuredContent';
 import useLocalize from '@hooks/useLocalize';
 import * as ReportActionsUtils from '@libs/ReportActionsUtils';
 import * as IOU from '@userActions/IOU';
 import * as Report from '@userActions/Report';
+import CONST from '@src/CONST';
+import type {ReportAction} from '@src/types/onyx';
 import BaseReportActionContextMenu from './BaseReportActionContextMenu';
+import type {ContextMenuType, ReportActionContextMenu} from './ReportActionContextMenu';
 
-function PopoverReportActionContextMenu(_props, ref) {
+type ContextMenuAnchorCallback = (x: number, y: number) => void;
+
+type ContextMenuAnchor = {measureInWindow: (callback: ContextMenuAnchorCallback) => void};
+
+type Location = {
+    x: number;
+    y: number;
+};
+
+function getPageCoords(event: GestureResponderEvent | MouseEvent): MouseEvent | NativeTouchEvent {
```

Comment:
The return type doesn't match with the name

Reply:
- Rename to `extractPointerEvent`
- Or switch to return type like `{readonly pageX: number, readonly pageY: number}`

Reply:
You are right, done!